junit/MatrixTest.java

```diff
@@ -71,11 +72,10 @@ public void tooShort(){
     //Matrix(double[] vals, int m){}
     
     /**
-     * testMatrixConstructor3 -Tests the third matrix constructor as specified in issue #7.
+     * testMatrixConstructor3a -Tests the third matrix constructor with valid parameters as specified in issue #7.
```

Comment:
The makefile I have is outdated. I honestly am a little confused on the branch structure right now. I need to merge your updated makefile from the makefile-update branch into this one to verify that checkstyle passes. Should I do this? @whereswaldon @bclinthall